README.md

```diff
@@ -35,25 +35,354 @@ After signing up for a Moesif account, your Moesif Application Id will be displa
 6. Go back to [Cloudflare Workers Dashboard](https://dash.cloudflare.com/workers) and select `Add Route`.
 * `Route` : Set a valid route. We recommend matching all requests for your domain. For example if your domain is `acmeinc.com`, your pattern should be `*acmeinc.com/*`.
 * `Worker`: Select the worker created above.
-* Save. 
+* Save.
 
 #### Try it!
 Just visit your link on cloudflare using your web browser `https://my-cloudflare-domain/my-path` or
 ```bash
 curl https://my-cloudflare-domain/my-path
 ```
-The API calls should show up in Moesif event stream. 
+The API calls should show up in Moesif event stream.
 
 ## [Alternative] Install via Cloudflare Workers (Custom install) using `wrangler` CLI
-Using `Cloudflare/Wrangler` CLI allows for automated install using `wrangler publish`, as well as view/`tail` using `wrangler tail` command 
-* Visit [WRANGLER.md](WRANGLER.md) for details.
+Using `Cloudflare/Wrangler` CLI allows for automated install using `wrangler publish`, as well as view/`tail` using `wrangler tail` command
+* Visit [WRANGLER.md](https://github.com/Moesif/moesif-cloudflare/blob/master/WRANGLER.md) for details.
```

Comment:
I think within github hyperlink markdown without full url works and links to current branch as relative path which is what we want? is this change addressing something? I dont think we will modify wrangler.md so its fine either ways.

Reply:
basically, our docs page just scrapes read me 
so here: 
 https://www.moesif.com/docs/server-integration/cloudflare-workers/#alternative-install-via-cloudflare-workers-custom-install-using-wrangler-cli

So either we have to copy everything into README, If you don't have absolute url, it will be a 404 error.